src/amo/components/GuidesAddonCard/styles.scss

```diff
@@ -0,0 +1,101 @@
+@import '~amo/css/styles';
+
+.GuidesAddonCard {
+  display: flex;
+  flex-direction: column;
+
+  // stylelint-disable max-line-length
+  // TODO: check on get-firefox button style.
+  // See https://github.com/mozilla/addons-frontend/issues/6432#issuecomment-433083079.
+  .Button--get-firefox,
+  .AMInstallButton {
+    @include respond-to(medium) {
+      align-self: center;
+      margin-left: auto;
+    }
+  }
+
+  .Icon-trophy {
+    height: 12px;
+    margin-right: 4px;
+    width: 12px;
+  }
+
+  .AddonTitle {
+    font-size: 16px;
+    font-weight: 500;
+    margin-bottom: 0;
+  }
+
+  .GuidesAddonCard-content-header-title {
+    .AddonTitle-author {
+      color: $grey-50;
+      font-size: 14px;
+      padding-top: 0;
+
+      a {
+        color: $blue-50;
+        font-size: inherit;
+        text-decoration: underline;
+      }
+    }
+  }
+
+  .Notice-error {
+    margin-bottom: 12px;
+  }
+}
+
+.GuidesAddonCard-content {
+  display: flex;
+  flex-wrap: wrap;
+
+  @include respond-to(medium) {
+    flex-wrap: nowrap;
+  }
+}
+
+.GuidesAddonCard-content-icon {
+  height: 40px;
+  margin-bottom: 12px;
+  width: 40px;
+}
+
+.GuidesAddonCard-content-header {
+  @include respond-to(medium) {
+    display: flex;
+  }
+}
+
+.GuidesAddonCard-content-header-title {
+  display: flex;
+  flex-direction: column;
+
+  @include respond-to(medium) {
+    margin: 0 16px;
+  }
+}
+
+.GuidesAddonCard-content-header-staff-pick {
+  color: $grey-50;
+  font-size: 10px;
```

Comment:
You can introduce a new font variable for this. Maybe call it `font-size-micro`? If you grep for 'font-size: 10px' you'll find a few other places that can share the same variable. Are you sure this matches the mock exactly? I can't seem to access the Sketch symbol for this part so I can't measure it.

Reply:
yeh I was eyeballing this as I didn't see font sizes in the mock. I was planning on showing @pwalm the display when I turned this all on and get feedback then. is that okay?

Reply:
Yes, that's fine since we don't currently have a variable here.